boards/shields/apollo4p_evb_disp_shield_rev2/apollo4p_evb_disp_shield_rev2.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2025, Ambiq Micro Inc. <www.ambiq.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		accel0 = &adxl362_test;
+	};
+};
+
+&spi1 {
```

Comment:
same comment - this will have to be something like apollo_evk_spi or something like that